### PR TITLE
Update specification.md

### LINKED ISSUE #8: azure blob storage connection parameter documentation
As stated in the title. update documentation with azure blob storage connection parameters information.

### PR DESCRIPTION
**Pull request by pj-spoelders**
Closes #8

### CODE REVIEW

**Comment by pj-spoelders** [comment]
Closes #8

**Review by morazow — APPROVED** [review]

**Comment by jakobbraun** [comment]
@pj-spoelders Please wait for the approval of all reviewers before merging

**Review by jakobbraun — APPROVED** [review]

**Review by exazg — APPROVED** [review]

**Review by redcatbear — APPROVED** [review]

**Pull request merged**
